samba/rootfs/usr/share/tempio/smb.gtpl

```diff
@@ -3,6 +3,17 @@
    workgroup = {{ .workgroup }}
    server string = Samba Home Assistant
 
+   {{ if .winsserver_mode }}
+   wins support = yes
+   {{ end }}
+
+   {{ if .masterbrowser_mode }}
+   domain master = yes
+   preferred master = yes
+   local master = yes
+   os level = 255
```

Comment:
From what I understand that is really the highest OS level there is. I get you want Home Assistant to be the  Maybe we should pick something slightly lower, e.g. 200? This would allow an other/external Samba instance to pick something higher if they want to be master browser.

```suggestion
   os level = 200
```